examples/capedocs.token

```diff
@@ -0,0 +1 @@
+eyJhbGciOiJSU0FTU0FfUFNTX1NIQV8yNTYiLCJ0eXAiOiJKV1QifQ.eyJhdWQiOlsiY2FwZS1hcGkiXSwiZXhwIjoxNzA2OTg0MDczLCJpYXQiOjE2NzU0NDgwNzMsImlzcyI6ImNhcGUtcHJpdmFjeSIsImp0aSI6IlRqeUhxOUdCZEZhOWRVR1FXY2VzSm8iLCJuYmYiOjE2NzU0NDgwNzMsInN1YiI6ImdpdGh1YnwxMTQ2OTY2MjIifQ.bMw_Xo3cMEiTHnNqr7mQaCtaGd0GOmTSqRQO8AXLOdAnJv4CeGw3kVwpYhJ9EZjG0opjlkX8JUzV1ph1eUjgUO1aFuHh-TvD-WMTZhZsbRojLC20WjHmrVfz5phm3Xter9bdroLuTFOwi2dpmjfyGqcoMZh4yQr_XU_ANHCIQ9OMBISgwQMVdDNYVUK0sL_byslX-ddHzt5dk9Tsnh3wvo6eKSY6hYpnh0cXl96PfCSzk6GtIZTTB_HRfHF6xuBP5zlazhT45um0i2KtGD5iKtFH1brbkWhejBf461NJ1ksePDzHvNRBR2KZEOKFVU-RsysQbao8rBpl3SasN3MP5Q
```

Comment:
Is this a PAT? If so should be a secret.

Reply:
ah gotcha, I was under the assumption that `capedocs` itself was meant for public consumption. i can revoke the token though. any suggestions on how to present this instead?

Reply:
I think unfortunately the user will need to generate their own PAT.

Reply:
set up a dedicated user for pycape docs, we can remove this when function-scoping is implemented for PATs

Reply:
Thanks!